UX/ui-service/MyServices/design.md

```diff
@@ -0,0 +1,44 @@
+# My Services
+* My Services are a primary navigation item.
+* This area is for the end user
+* This area is provided so the user can view the current state and details of their services.
+
+## View My Services
+* A List View is the default view for My Services and it will not show all fields in this view.
+* A Table View is the secondary view that should be added once the PatternFly Angular Table is available. The table will show additional fields that may be missing from the List View.
+* Bulk actions can be accessed through the button groups at the top of the list.
+
+### List View
+![Image of List View.](img/Services-ListView.png)
+* The Service Names are links and will bring the user to a service details page.
+* The exact List View fields to be included are TBD.
+* **Implementation Details:** Reverence the Angular PatternFly Expandable List View Component.
+
+### Detail View
```

Comment:
@serenamarie125 Yes, I changed the format to show there is a Details page which includes a summary View (shown) and in the future a dashboard and topology. Also changed the order to come after expanded list. Let me know what you think.